src/components/MultiSelect/readme.md

```diff
@@ -0,0 +1,176 @@
+##### MultiSelect base
+
+```js
+import React, { useState } from 'react';
+import { MultiSelect, Option } from 'react-rainbow-components';
+
+const containerStyles = {
+    maxWidth: 400,
+};
+
+const MultiSelectExample = props => {
+    const [value, setValue] = useState([]);
+
+    const handleChange = val => {
+        setValue(val);
+    }
+
+    return (
+        <MultiSelect
```

Comment:
The example should be work as a real use case

Reply:
sorry, just had to update the branch with the InternalDropdown multiple changes, examples should work as expected now